sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -112,8 +131,31 @@ public HttpPipeline getHttpPipeline() {
             });
     }
 
-    // TODO: This is a temporary implementation for sample purposes. This should be spruced up/replaced once this API is actually designed.
-    // Input is Object and output is Response<T>.
+    /**
+     * Creates a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin.
+     * @param digitalTwin The application/json digital twin to create.
+     * @param clazz The model class to convert the response to.
+     * @param <T> The generic type to convert the response to.
+     * @return The application/json digital twin created.
```

Comment:
How about "The deserialized application/json object representing the digital twin"?

Reply:
Here and everywhere for this comment and below. We need to avoid using the phrase "convert" when we should be saying "deserialize"

Reply:
a string representation of a json object isn't deserialized though, right? If anything, it is serialized from a class to this string.

Reply:
correct, that would be string representation